[DOCS] Event filters and Host isolation exceptions by Policy (#1615)
elastic/security-docs: docs/management/admin/host-isolation-exceptions.asciidoc

Host isolation exceptions support IPv4 addresses, with optional classless inter-domain routing (CIDR) notation.

> By default, a host isolation exception is recognized globally across all hosts running {endpoint-sec}. If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], you can also assign a host isolation exception to a specific {endpoint-sec} integration policy, affecting only the hosts assigned to that policy.

Reviewer: Slight difference with the Host Isolation exceptionlist artifact - the entire host isolation feature is platinum/enterprise, so even having a global list is in a paid tier. 

(Realizing we don't say on the Host Isolation page that it is a paid feature, so that may be good to add separately!)

.. `Name your host isolation exceptions`: Enter a name to identify the host isolation exception.
.. `Description`: Enter a description to provide more information on the host isolation exception (optional).
.. `Enter IP Address`: Enter the IP address for which you want to allow communication with an isolated host. This must be an IPv4 address, with optional CIDR notation (for example, `0.0.0.0` or `1.0.0.0/24`, respectively).

> . Select an option in the *Assignment* section to assign the host isolation exception to a specific integration policy:

Reviewer: I know you mentioned it above, but may also want to add a platinum/enterprise note here

Author: There's a note below, which we can move up to the entire step.

Reviewer 2: I think it makes sense to just mention that the entire feature is platinum and we can remove the other references.  Thoughts @caitlinbetz ?

Reviewer: oh yes that makes sense. Sorry, I think I meant to comment this on the event filters page. This looks good to me.

+
* `Global`: Assign the host isolation exception to all integration policies for {endpoint-sec}.

> * `Per Policy` (Platinum or Enterprise subscription only): Assign the host isolation exception to one or more specific {endpoint-sec} integration policies. Select each policy where you want the host isolation exception to apply.

Author: ```suggestion
* `Per Policy`: Assign the host isolation exception to one or more specific {endpoint-sec} integration policies. Select each policy where you want the host isolation exception to apply.
```

> * `Per Policy` (Platinum or Enterprise subscription only): Assign the host isolation exception to one or more specific {endpoint-sec} integration policies. Select each policy in which you want the host isolation exception to be applied.

Reviewer 3: ```suggestion
* `Per Policy` (Platinum or Enterprise subscription only): Assign the host isolation exception to one or more specific {endpoint-sec} integration policies. Select each policy where you want the host isolation exception to apply.
```

+

> NOTE: You can also select the `Per Policy` option without immediately assigning a policy to the host isolation exception. For example, you could do this to create and review your host isolation exception configurations before putting them into action with a policy.

Reviewer 4: ```suggestion
NOTE: You can also select the `Per Policy` option without immediately assigning a policy to the host isolation exception. For example, you could do this to create and review your host isolation exception configurations before activating them with a policy.
```

Reviewer 4: Only a rec. Feel free to ignore if it's wrong or you decide against using it!

> NOTE: You can also select the `Per Policy` option without assigning a policy to the host isolation exception at this time. For example, you could do this if you want to create and review your host isolation exception configurations before putting them into action with a policy.

Reviewer 3: ```suggestion
NOTE: You can also select the `Per Policy` option without immediately assigning a policy to the host isolation exception. For example, you could do this to create and review your host isolation exception configurations before putting them into action with a policy.
```

. Click **Add Host isolation exception**. The new exception is added to the *Host isolation exceptions* list.

[discrete]
[[manage-host-isolation-exceptions]]
== View and manage host isolation exceptions

> The **Host isolation exceptions** list displays all the host isolation exceptions that have been configured for {elastic-sec}. To refine the **Host isolation exceptions** list, enter a query in the search bar. You can search by name, description, or IP address.

Reviewer 4: This might be common knowledge, but I was wondering whether it'd be worth specifying which query lang the user should use when searching for the exception. For example, would they enter an EQL query into the search bar?

Author: Maybe "enter a query" isn't quite right here -- it's really just a simple search. AFAIK you can't build sophisticated queries like you can in other areas of the app. @nastasha-solomon how about something like this?
```suggestion
The **Host isolation exceptions** list displays all the host isolation exceptions that have been configured for {elastic-sec}. To refine the **Host isolation exceptions** list, use the search bar to search by name, description, or IP address.
```

[role="screenshot"]
image::images/host-isolation-exceptions-ui.png[List of host isolation exceptions]

[discrete]
[[edit-host-isolation-exception]]
=== Edit a host isolation exception

> You can change the configuration of each host isolation exception. If you have a Platinum or Enterprise subscription, you can also change the policy assigned to a host isolation exception.

Reviewer 3: ```suggestion
You can individually configure each host isolation exception. If you have a Platinum or Enterprise subscription, you can also change the policy or policies assigned to each host isolation exception.
```
Are policies assigned to exceptions or exceptions assigned to policies? I'm inclined to think it's the latter, but not sure how much it matters.
